tools/run_tests/run_tests_matrix.py

```diff
@@ -181,7 +181,17 @@ def _create_portability_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS)
   # portability C and C++ on x64
   for compiler in ['gcc4.4', 'gcc4.6', 'gcc5.3',
```

Comment:
After talking with Craig, we thought it wouldn't make a difference if we changed from 4.6 to 4.8 for C tests, so I decided to not change it. I would be open to changing it if it was better for testing though.